src/main/kotlin/com/katanox/tabour/config/EventPollerProperties.kt

```diff
@@ -43,13 +43,11 @@ data class EventPollerProperties(
     var batchSize: Int = DEFAULT_BATCH_SIZE,
 
     /**
-     * The number of threads that should poll for new messages. Each of those threads will poll a
-     * batch of batchSize messages and then wait for the pollDelay interval until polling the next
-     * batch.
+     * The number of coroutines that should poll for new messages and then consume them.
      *
-     * The default is 1.
+     * The default is 50.
      */
-    var pollingThreads: Int = 1,
+    var pollingCoroutines: Int = 50,
```

Comment:
i think we should not define the type of poller in it's name.

`numOfPollers` should be enough as it "hides" the implementation

Reply:
Good suggestion